js/components/items-container.tsx

```diff
@@ -354,7 +354,7 @@ class ItemsContainer extends React.Component<IComponentProps, IItemsContainerSta
             }
 
             // Sort the filtered list before slicing
-            if (this.state.sortedBy) {
+            if (this.state.sortedBy && Array.isArray(objectData)) {
```

Comment:
Should this `if` also cover empty arrays?

Reply:
It's not really necessary - sorting an empty array gives and empty array.